src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,49 @@
 package core.basesyntax;
 
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.stream.Stream;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String result = generateReportFromDataFile(fromFileName);
+        writeData(toFileName, result);
+    }
+
+    private void writeData(String toFileName, String result) {
+        Path outputFile = Paths.get(toFileName);
+        try {
+            Files.write(outputFile, result.getBytes(StandardCharsets.UTF_8));
+        } catch (IOException ioException) {
+            throw new RuntimeException(ioException.getMessage());
```

Comment:
Don't ignore exceptions: Instead of just throwing the exception message, re-throw a RuntimeException with the original exception and some message. This provides more context and preserves the stack trace.

Reply:
I don't need it in this task.